common/app/views/main.scala.html

```diff
@@ -25,6 +25,8 @@
             ("childrens-books-site", metaData.section == "childrens-books-site")))"
         itemscope itemtype="http://schema.org/WebPage">
 
+        <div class="win-chrome-font-fix">@* hack to fix fonts in win/chrome with font-smoothing disabled *@</div>
```

Comment:
only chrome seems to be affected now - i think all the hinting etc stuff you did has everything else covered?